openmdao/core/tests/test_src_indices.py

```diff
@@ -106,14 +106,11 @@ def test_multiple_inputs_different_src_indices(self):
         assert_near_equal(p.model.get_val('a'), 99.)
         assert_near_equal(p.model.get_val('b'), 2.)
         assert_near_equal(p['g1.y'], 101.)
-        assert_near_equal(p['g1.a'], 99.)
```

Comment:
Wow, we actually had a test with the hybrid names intentionally added to it.

Reply:
Yeah I think we actually had multiple tests for hybrid names.